chia/wallet/nft_wallet/nft_info.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import dataclass
+from typing import List
+
+from chia.util.ints import uint64
+from chia.util.streamable import Streamable, streamable
+
+
+@streamable
+@dataclass(frozen=True)
+class NFTInfo(Streamable):
+    """
+    NFT Info
+    """
+
+    launcher_id: str
+    """
+    Launcher coin ID
+    """
+    nft_coin_id: str
+    """
+    Current NFT coin ID
+    """
+    did_owner: str
+    """
+    Owner DID
+    """
+    royalty: uint64
+    """
+    Percentage of the transaction fee paid to the author
```

Comment:
I might mention that the royalty is represented as the percentage * 1000, so 1500 == 1.5%